TODO: Fix this:

aws_instance.ec2_mongodb-router (remote-exec): 	"errmsg" : "in seed list shard2/52.207.211.247:27017, host 52.207.211.247:27017 does not belong to replica set shard2; found { topologyVersion: { processId: ObjectId('62c6d3edd1c5c96498138837'), counter: 7 }, hosts: [ \"ip-172-31-94-76:27017\" ], setName: \"shard2\", setVersion: 1, ismaster: true, secondary: false, primary: \"ip-172-31-94-76:27017\", me: \"ip-172-31-94-76:27017\", electionId: ObjectId('7fffffff0000000000000001'), lastWrite: { opTime: { ts: Timestamp(1657197607, 1), t: 1 }, lastWriteDate: new Date(1657197607000), majorityOpTime: { ts: Timestamp(1657197607, 1), t: 1 }, majorityWriteDate: new Date(1657197607000) }, isImplicitDefaultMajorityWC: true, maxBsonObjectSize: 16777216, maxMessageSizeBytes: 48000000, maxWriteBatchSize: 100000, localTime: new Date(1657197617679), logicalSessionTimeoutMinutes: 30, connectionId: 7, minWireVersion: 0, maxWireVersion: 13, readOnly: false, compression: [ \"snappy\", \"zstd\", \"zlib\" ], ok: 1.0, $clusterTime: { clusterTime: Timestamp(1657197607, 1), signature: { hash: BinData(0, 0000000000000000000000000000000000000000), keyId: 0 } }, operationTime: Timestamp(1657197607, 1) }",